trajectory_executor/src/trajectory_executor_node.cpp

```diff
@@ -55,8 +55,8 @@ namespace trajectory_executor
       out[config_.default_control_plugin] = config_.default_control_plugin_topic;
 
       //Hardcoding platoon control plugins
-      std::string control_plugin2 = "PlatooningControlPlugin";
-      std::string control_plugin_topic2 = "/guidance/PlatooningControlPlugin/plan_trajectory";
+      std::string control_plugin2 = "platoon_control";
```

Comment:
I think plugin isn't needed in the node name since its in the namespace. As far as the package name goes, the main ones lacking it are the platooning plugins which I'm hesitant to change right now to avoid more merge conflicts or new bugs with the IHP2 release when its synced back.